apps/sms/services/activity/client.js

```diff
@@ -0,0 +1,62 @@
+/* global EventDispatcher,
+          threads
+*/
+
+(function(exports) {
+  'use strict';
+
+  /**
+   * Name of the service that is responsible for managing activities.
+   * @type {string}
+   */
+  const SERVICE_NAME = 'activity-service';
+
+  /**
+   * Definition of all available private members.
+   * @type {Object}
+   */
+  const priv = Object.freeze({
```

Comment:
Yeah, there is also [1] to always get the same symbol for the name. Haven't tried it though

[1] https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Symbol/for